packages/api/test/get-repoorgs-test.js

```diff
@@ -95,31 +90,33 @@ describe('Getting Repos and Orgs', () => {
   });
 
   it('limit search (only on repos)', async () => {
-    const resp = await fetch('http://localhost:3000/api/repo?startswith=z&limit=3');
+    const resp = await fetch('http://localhost:3000/api/repo?org=bigorg&startswith=z&limit=3');
     const respText = await resp.text();
     assert.strictEqual(JSON.parse(respText).length, 3);
   });
 
-  it('should be able to search by org/beginning of name', async () => {
-    const querySuffixes = ['bigor', 'bigorg', 'bigorg%2F', 'bigorg%2Fz'];
-    for (let t = 0; t < querySuffixes.length; t++) {
-      const resp = await fetch('http://localhost:3000/api/repo?startswith=' + querySuffixes[t]);
-      const respText = await resp.text();
-      assert.strictEqual(JSON.parse(respText).length, 4);
-    }
-  });
-  it('should return all with no parameter', async () => {
-    const resp = await fetch('http://localhost:3000/api/repo');
+  // deprecated feature:
+  // it('should be able to search by org/beginning of name', async () => {
```

Comment:
I tend to be pretty cavalier about deleting code, rather than commenting it out (_the git history can always be referenced if we want to look at an old feature._).